test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -1116,6 +1136,8 @@ class test_uct_sockaddr_multiple_cms : public test_uct_sockaddr {
                        uct_ep_disconnect_cb_t disconnect_cb,
                        void *user_data)
     {
+        entity::scoped_async_lock listen_lock(*m_server);
+        scoped_async_lock accept_lock(*m_test_async);
```

Comment:
`entity::scoped_async_lock` ?

Reply:
no, the test has own async context outside of entities

Reply:
why need 2 types of async lock?

Reply:
related to migration conn_request between CMs, first one for origin entity second one the test specific

Reply:
then let's make a more generic async_lock, outside of entity, instead of uct_test::entity::scoped_async_lock.
it will have 2 constructors: 1. with entity& 2. with async*